README.md

```diff
@@ -132,36 +132,28 @@ from: https://github.com/openssl/openssl
 The instruction for Windows:
 http://developer.covenanteyes.com/building-openssl-for-visual-studio/
 
-2. Compile and install Pthreads for Windows from this submodule:
+2. Download the pre-compiled Pthreads library for Windows via NuGet:
```

Comment:
The original guide should be retained. We need to come up with a separate document to provide several options to build the library on Windows.

Reply:
Except this message was removed by you in referenced PR 814... so it's actually you that removed this and I just copied you because you highlighted the conflict with PR814...

Reply:
Blame Roman, he's the one that started out pointing out that README would be outdated!

For me, this one falls under 'no good deed goes unpunished'.

And, never let your PR get too big :)

In a few moments, the readme will just be reverted to master.